aicsimageio/aics_image.py

```diff
@@ -478,10 +478,7 @@ def get_physical_pixel_size(self, scene: int = 0) -> Tuple[float]:
         sizes: Tuple[float]
             Tuple of floats representing the pixel sizes for X, Y, Z, in that order.
```

Comment:
On this note, would it be breaking to alter the order of this now to be consistent with the rest of the project? It seems like that would be done on CZIReader and OmeTiffReader

Reply:
I assume @toloudis would strongly advise against breaking changes.

Reply:
Not only that but I could easily argue that this XYZ is not really the same thing as the dimension order of the ndarray.  No one will ever care to transpose these values.  They just need to know what order they come in.

Reply:
I do concede that it can be confusing to have z first sometimes and last other times, but it should be well documented :)

Reply:
Got it 😁